hoomd/mpcd/integrate.py

```diff
@@ -214,9 +219,8 @@ def cell_list(self):
     @property
     def virtual_particle_fillers(self):
         """Sequence[hoomd.mpcd.fill.VirtualParticleFiller]: Solvent
-            virtual-particle fillers.
-
-        """
+        virtual-particle fillers.
+        """  # noqa: D205,D415
```

Comment:
I kept getting a ton of flake8 errors for having the property description broken across multiple lines, although this should work with Google-style docstrings. I added noqa on those errors, but let me know if you have a better idea!

Reply:
I would prefer not to add noqa for all these. You can continue the line with backspace:
https://github.com/glotzerlab/hoomd-blue/blob/da678feb649f61632a74ceca7f52aa9ab6db49a3/hoomd/md/force.py#L79-L80

Reply:
Ah, gotcha! This fixed the problem, and I removed noqa.